### PR TITLE
Levels

### PR DESCRIPTION
**Pull request by JerPk**

### CODE REVIEW

File: src/main/java/spaceinvaders/spaceinvaders_framework/Game.java [hunk 1]
```diff
@@ -76,10 +79,10 @@
...
      * Vector to store all alien, bullet and barrier objects
      */
-    private Vector<Alien> aliens;
-    private Vector<Bullet> alienBullets;
-    private Vector<Bullet> shipBullets;
-    private Vector<Barrier> barriers;
+    public Vector<Alien> aliens;
+    public Vector<Bullet> alienBullets;
+    public Vector<Bullet> shipBullets;
+    public Vector<Barrier> barriers;
```
Comment (Mari3) [inline src/main/java/spaceinvaders/spaceinvaders_framework/Game.java]: It's not a good idea to make all these vectors public. That makes the game class head towards a god class again and violates encapsulation rules.

Comment (JerPk) [inline src/main/java/spaceinvaders/spaceinvaders_framework/Game.java]: All the vectors get cleared when a new level is created and the aliens and barriers get created in the Level class. So need access to them ...

**Comment by JerPk** [comment]
The vectors are private again.

**Comment by harveyvanveltom** [comment]
Looks great. I believe it is ready to be merged with master.

**Pull request merged**